src/views/About.vue

```diff
@@ -188,6 +190,7 @@ img {
     max-width: 50%;  /* Sets the maximum width to 50% of its container */
     height: auto;    /* Maintains the aspect ratio of the image */
 }
-
-  </style>
-  
\ No newline at end of file
+.top-image {
+    max-width: 100% !important;  /* Set the maximum width to 100% of its container */
```

Comment:
The use of important is okay in this case but you should attempt other means of styling and formatting before falling back to it as its not the most recommended method.